app/imgprep/imgprep.py

```diff
@@ -35,7 +37,7 @@ def get_sample_data_array(self, img, file_result=None):
 
         # carefully extract sample if there's at least one `True`
         if upper[0] != -1:
-            img = img[lower[0] : upper[0], lower[1] : upper[1]]
+            img = img[lower[0]: upper[0], lower[1]: upper[1]]
```

Comment:
Won't agree with you on this point: it's not a key-value pairs, it's numpy slice indices:

https://docs.scipy.org/doc/numpy/reference/arrays.indexing.html#combining-advanced-and-basic-indexing

So I will agree on providing (or avoiding as well) single space from both sides of colon simultaneously - your variant looks definetely like key-value pairs.